Lib/typing.py

```diff
@@ -1395,9 +1395,26 @@ def _get_protocol_attrs(cls):
     return attrs
 
 
-def _is_callable_members_only(cls):
+def _is_callable_or_classvar_members_only(cls):
+    attr_names = _get_protocol_attrs(cls)
+    annotations = getattr(cls, '__annotations__', {})
     # PEP 544 prohibits using issubclass() with protocols that have non-method members.
-    return all(callable(getattr(cls, attr, None)) for attr in _get_protocol_attrs(cls))
+    for attr_name in attr_names:
+        attr = getattr(cls, attr_name, None)
+        if callable(attr):
+            continue
+        annotation = annotations.get(attr_name)
+        if getattr(annotation, '__name__', None) == 'ClassVar':
+            continue
```

Comment:
```suggestion
        if getattr(annotation, '__origin__', None) is ClassVar:
            continue
```

Reply:
I'd considered that but decided not to. It won't work if typing is re-imported as a fresh module multiple times.